packages/feedback/src/app/modules/feedback/feedback-item/feedback-item.component.sass

```diff
@@ -0,0 +1,123 @@
+@import "../../../../sass/component/_component.sass"
```

Comment:
Can you convert this file to the `scss` syntax? I think we discussed to use the new scss syntax for all the SASS files.

Do the same for any other similar files.

Reply:
I am new to CSS it will take me long time to convert it because there are other file as well.
will talk about it in stand up.